### PR TITLE
Update and cleanup styles

### PR DESCRIPTION
**Pull request by sbastn**
This PR updates the styles and cleans old assets. In the process, I've made this site closer to the support site by adding CTAs to the app.

Highlights:
- Adds CTAs to app. They are currently going to our support campaign page where we explain a bit more how we can help developers with support.
- Uses same typography and layout as the support site
- Remove old bootstrap version

Each individual commit should help navigate all the changes that were made.

## 👀 Preview

Screenshot_2021-01-07 DNSimple Developer

## 🔍 QA

Browse around. Compare with live version. Layout might be slightly different to match the support site. CTAs should point to the support campaign page.

### CODE REVIEW

File: content/assets/css/_typography.scss [hunk 1]
```diff
@@ -23,26 +31,58 @@ h1 {
...
+}
+
+hr {
+  margin: 20px 0;
+  border: 0;
+  border-top: 1px solid #eee;
+}
+
+code {
+  padding: 2px 4px;
```
Comment (DXTimer) [inline content/assets/css/_typography.scss:67]: This is causing a visual bug, where the content is no longer aligned across the body of the code. Please refer to the screenshot.

Screenshot 2021-01-08 at 9 23 31

Comment (sbastn) [reply content/assets/css/_typography.scss:67]: Great find! This is fixed!
Screen Shot 2021-01-08 at 8 40 29 AM

**Review by DXTimer — APPROVED** [review]
💯 :shipit:

File: layouts/header.html [hunk 2]
```diff
@@ -11,7 +11,6 @@
 
   <link rel="shortcut icon" type="image/png" href="/assets/images/favicon.ico" />
 
-  <link rel="stylesheet" type="text/css" href="/assets/bootstrap/css/bootstrap.css">
```
Comment (dallasread) [inline layouts/header.html:14]: 🥳

**Review by dallasread — APPROVED** [review]
👍 Nicely done!

**Pull request merged**